html_css/grid-lessons/advanced_grid_properties.md

```diff
@@ -85,7 +85,7 @@ Check it out for yourself!
 
 ### Fractional units
 
-Now that we know how to quickly create many grid tracks, it's time to learn how to start making them _dynamic_. Dynamic, in this context, simply means "flexible" or "responsive in some way." The opposite of dynamic is _static_, or fixed at a certain defined height, like `150px`, which we used in the setup of this sample grid.
+Now that we know how to quickly create many grid tracks, it's time to learn how to start making them _dynamic_. Dynamic, in this context, simply means "flexible" or "responsive in some way". The opposite of dynamic is _static_, or fixed at a certain defined height, like `150px`, which we used in the setup of this sample grid.
```

Comment:
This one is actually not a grammatical error. The punctuation goes outside the quotation here and the original content was correct.

Reply:
yeah, agreed, it was just an punctuation error, is there anything in particular that needs to be done before proceeding?

thanks :)

Reply:
Brain fart on my end! I meant to say the punctuation goes inside the quotation like it was originally. My mistake for saying "outside" in that comment :-)

Reply:
apparently we both were wrong about that, in different ways, i'll revert back to original then, thanks :)